WebRoot/js/zimbraMail/core/ZmOperation.js

```diff
@@ -145,7 +145,6 @@ function() {
     ZmOperation.registerOp(ZmId.OP_GROUPBY_PRIORITY, {textKey:"groupByPriority"});
     ZmOperation.registerOp(ZmId.OP_GROUPBY_SIZE, {textKey:"groupBySize"});
     ZmOperation.registerOp(ZmId.OP_GROUPBY_TAG,  {textKey:"groupByTag"});
-	ZmOperation.registerOp(ZmId.OP_GO_TO_URL, {image:"URL", textKey:"goToUrlAlt"});
```

Comment:
should we remove associated translations as well?

Reply:
removed `goToUrlAlt` from ZmMsg properties files.